src/OrbitGl/GlSlider.cpp

```diff
@@ -130,13 +151,9 @@ void GlSlider::DrawBackground(GlCanvas* canvas, float x, float y, float width, f
   batcher.AddBox(bar_box, bar_color_, shared_from_this());
 }
 
-void GlSlider::DrawSlider(GlCanvas* canvas, float x, float y, float width, float height,
-                          ShadingDirection shading_direction) {
-  Batcher& batcher = canvas->GetBatcher();
-  Color color =
-      canvas->GetPickingManager().IsThisElementPicked(this) && drag_type_ == DragType::kPan
-          ? selected_color_
-          : slider_color_;
+void GlSlider::DrawSlider(Batcher& batcher, float x, float y, float width, float height,
+                          ShadingDirection shading_direction, bool is_picked) {
+  Color color = is_picked && drag_type_ == DragType::kPan ? selected_color_ : slider_color_;
```

Comment:
Nit: Could you put parentheses around the condition? To me this would make it easier to read.

Reply:
This is no longer the case in the latest version, won't fix it in the intermediate commit.